server/views/pages/bookedtoday/arrivals/autoMatchingRecords/singleMatchingRecordFound.njk

```diff
@@ -1,39 +1,24 @@
 {% extends "../../../../partials/layout.njk" %}
-{% from "../../../../components/prisoner-summary-detail.njk" import prisonerSummaryDetail %}
+{% from "../../../../components/prisoner-split-view.njk" import prisonerSplitView %}
 
 {% block content %}
 
     <div class="govuk-grid-row">
         <div class="govuk-grid-column-two-thirds">
             <span class="govuk-caption-xl"> Confirm a prisoner's arrival </span>
             <h1 class="govuk-heading-l govuk-!-margin-bottom-7">This person has an existing prisoner record</h1>
-            <p> {{ arrival.firstName }} {{ arrival.lastName }} will be booked into the establishment using their existing prisoner record.</p>
+            <p> {{ arrival.firstName }} {{ arrival.lastName }} will be booked into the establishment using their
+                existing prisoner record.</p>
         </div>
     </div>
-    <div class="govuk-grid-row govuk-!-padding-left-3 govuk-!-margin-top-4 govuk-!-margin-bottom-8 flex" >
-        <div class="column-width-40 govuk-!-padding-right-9">
-            <h2 class="govuk-heading-m">Details from Person Escort Record</h2>
-            <div class = "prisoner-record-detail prisoner-head">
-                {{ prisonerSummaryDetail(arrival, "per-record" ) }}
-            </div>
-        </div>
-        <div class=" border-left-grey govuk-!-padding-left-9 column-width-60">
-            <h2 class="govuk-heading-m">Details from existing prisoner record</h2> 
-            <div class="prisoner-record-detail flex">
-                {{ prisonerSummaryDetail(data, "existing-record") }}
-                <img src="{{ '/prisoners/' + data.potentialMatches[0].prisonNumber + '/image' if data.potentialMatches[0].prisonNumber else '/assets/images/placeholder-image.png' }}" 
-                        class="prisoner-image-transfer govuk-!-margin-top-2" 
-                        alt="{{ data.potentialMatches[0].lastName }}, {{ data.potentialMatches[0].firstName }}" 
```

Comment:
think this might have been a bug, doesn't look like we pass a potentialMatch to the view?

Reply:
good spot